demo/linux_app/Dockerfile_linux_app

```diff
@@ -0,0 +1,37 @@
+# BUILDER
+FROM ubuntu:22.04
+
+# build ARGs
+ARG _LINUX_GIT_REPO='https://github.com/torvalds/linux.git'
+ENV LINUX_GIT_REPO=$_LINUX_GIT_REPO
+ARG _LINUX_GIT_BRANCH="v6.6"
+ENV LINUX_GIT_BRANCH=$_LINUX_GIT_BRANCH
+ENV LINUX_APP_DIR="/app"
+
+WORKDIR $LINUX_APP_DIR
+
+# install build dependecies
+RUN apt-get -y update
```

Comment:
Combine lines 14-18 as follows and remove the additional 23-24 apt get below

```
RUN apt-get -o APT::Retries=3 update -y && \
    apt-get upgrade -y && \
    apt-get -o APT::Retries=3 install -y --no-install-recommends \
        autoconf bc bison build-essential dkms flex gawk git libelf-dev libiberty-dev \
        libncurses-dev libpci-dev libssl-dev libudev-dev llvm openssl tzdata \
        && \
    apt-get -y autoremove && \
    apt-get -y clean && \
    rm -rf /var/lib/apt/lists/*
```

Reply:
integrated with some additional updates.